client/App.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render } from 'react-dom';
+import axios from 'axios';
+import ImageView from './ImageView';
+import ImageViewList from './ImageViewList';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: [],
+      selectedImage: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.fetcher = this.fetcher.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetcher();
+  }
+
+  fetcher() {
+    const id = Math.floor(Math.random() * 299);
+    axios.get(`http://localhost:3000/item/${id}/images`)
```

Comment:
When you get the chance, you definitely want to refactor this out of the component, and put it in a separate file.